tools/pd-ctl/pdctl/command/scheduler.go

```diff
@@ -423,14 +424,22 @@ func newConfigHotRegionCommand() *cobra.Command {
 		Short: "balance-hot-region-scheduler config",
 		Run:   listSchedulerConfigCommandFunc,
 	}
+
+	// Deprecated: 'list' command will be deprecated in future version, use 'show' command instead.
 	c.AddCommand(&cobra.Command{
 		Use:   "list",
+		Short: "list the config item (will be deprecated in feature version, use show command instead)",
+		Run:   listSchedulerConfigCommandFunc,
```

Comment:
How about adding `Deprecated`?

Reply:
Something like this?
`"list the config item (Deprecated: list command will be deprecated in future version, use show command instead.)"`

Reply:
`cobra.Command` has a `Deprecated` field and you can refer to `NewStoresCommand()`.

Reply:
I see, but I have few concerns using `Deprecated` field at the moment:
- The `list` command has not been deprecated yet, most likely will be in next major release which means the `Deprecated` field should be added then
- The command with `Deprecated` field will be hidden which indicates an incorrect impression to users.
  ``` 
  Available Commands:
    set         set the config item
    show        list the config item
  ```
- The command with `Deprecated` field will print extra message to screen which could break script if it is only expect the json output.
  ```
  Command "list" is deprecated, use show instead
  ```

Reply:
Ok, SGTM. First, we should change the document to let users know when we going to make it deprecated. Then we can mark it deprecated.

Reply:
Yes, that is why the short description contains the message `list command will be deprecated in future version`. I will create an issue to update the document.